db.js

```diff
@@ -424,37 +424,41 @@ var loans = (function () {
         return Promise.reject();
       } else {
         var insertQuery =
-          "INSERT INTO loans book_id, requester_contact, owner_contact, status) " +
-          "VALUES (?,?,?,?)";
-        var insertParams = [
-          loan.book_id,
-          loan.requester_contact,
-          " ",
-          "pending",
-        ];
+          "INSERT INTO loans (book_id, requester_id, owner_contact, status) " +
+          "VALUES (?,?,?,?,?)";
+        var insertParams = [loan.book_id, loan.requester_id, " ", loan.status];
         insertQuery = mysql.format(insertQuery, insertParams);
         return pool.query(insertQuery);
       }
     },
 
-    updateLoanById: function (loan) {
-      var sql =
-        "UPDATE loans SET book_id = ?, requester_contact = ?, owner_contact = ?, status = ? " +
-        "WHERE id = ?";
-      var inserts = [
-        loan.book_id,
-        loan.requester_contact,
-        " ",
-        loan.status,
-        loan.id,
-      ];
-      updateQuery = mysql.format(updateQuery, inserts);
+    updateLoanById: async function (loan) {
+      if (loan.status === "accepted") {
```

Comment:
With this restriction in place, how would you ever change a loan to "loaned" or "late"?

Reply:
I was thinking of that yesterday. My plan was to merge this PR as it has its own distinct card attached to it, and then buff out the rest of the loans routes with the appropriate logic for the other tasks required of them. There's another card in Jira called "fix loans for real," so I inferred that those other tasks fell within the realm of that card.

Reply:
Okay, sounds good. I'm fine with that plan. I'm most interested in getting the loans route fixed asap, and if you have a plan to make that happen, I'm all for it.

Reply:
Cool, I'll go ahead and merge